src/sentry/models/projectredirect.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import Model, FlexibleForeignKey
+
+
+# TODO: Should this be ProjectSlugHistory
```

Comment:
imo if it should be now is the only time you get to change it

Reply:
9a8d3c0 I'm fine with `ProjectRedirect`.